apps/storybook/src/ProgressBar.stories.tsx

```diff
@@ -0,0 +1,51 @@
+import type { Meta, StoryObj } from "@storybook/react";
+
+import { Flex, ProgressBar } from "@optiaxiom/react";
+import React from "react";
+
+const meta: Meta<typeof ProgressBar> = {
+  component: ProgressBar,
+  title: "Primitives / ProgressBar",
```

Comment:
I think this should go under `Feedback`
```suggestion
  title: "Feedback / ProgressBar",
```

Reply:
Addressed!